packages/dotlottie-js/src/common/lottie-animation-common.ts

```diff
@@ -57,6 +61,8 @@ export class LottieAnimationCommon {
     this._loop = options.loop ?? false;
     this._autoplay = options.autoplay ?? false;
     this._defaultActiveAnimation = options.defaultActiveAnimation ?? false;
+    this._hover = options.hover ?? false;
+    this._intermission = options.intermission ?? 0;
```

Comment:
don't we need to perform runtime validation on the `intermission`, to validate if it fits in the constraints, like being a non negative number  ?